metrics/config_observability.go

```diff
@@ -77,6 +80,7 @@ func NewObservabilityConfigFromConfigMap(configMap *corev1.ConfigMap) (*Observab
 		cm.AsString("logging.revision-url-template", &oc.LoggingURLTemplate),
 		cm.AsString("logging.request-log-template", &oc.RequestLogTemplate),
 		cm.AsBool("logging.enable-probe-request-log", &oc.EnableProbeRequestLog),
+		cm.AsBool("logging.enable-request-logging", &oc.EnableRequestLogging),
```

Comment:
should we use `logging.enable-request-log` to be consistent with `logging.enable-probe-request-log`?

Reply:
@yanweiguo could be done.